packages/ui/src/lib/settingsNavItem/SettingsNavItem.svelte

```diff
@@ -46,6 +47,10 @@ function click(): void {
   expanded = !expanded;
   onClick();
 }
+
+function isIconDefinition(icon: IconDefinition | SvelteComponent): icon is IconDefinition {
```

Comment:
This is something we are using in multiple place (from the top of my memory) might be nice to have a `icon-utils.ts` file somewhere